tidepods/cli.py

```diff
@@ -42,8 +42,8 @@ def s2(**kwargs):
 
     Example use:
 
-    tidepods s2 -s A:/ANSU/6_Tasks/_SDB_Tidepods/tidepods-ansu_TestFile/S2/S2B_MSIL1C_20210307T074749_N0209_R135_T37QDE_20210307T100607.SAFE 
-    -l MSL -o A:/ANSU/6_Tasks/_SDB_Tidepods/tidepods-ansu_TestFile/S2
+    tidepods s2 -s A:/user/6_Tasks/_SDB_Tidepods/tidepods-ansu_TestFile/S2/S2B_MSIL1C_20210307T074749_N0209_R135_T37QDE_20210307T100607.SAFE 
```

Comment:
it would be nice if you can change the examples here to something even more generic, like `C:/path-to-sentinel-folder` and output `C:/path-to-output-folder`.